.github/workflows/dockerhub.yml

```diff
@@ -26,52 +27,44 @@ jobs:
       run:
         shell: bash
     steps:
+      - name: Check Git event trigger
```

Comment:
Action will push to tag matching PR base branch as currently written for PRs. Since we do not want to push docker images for PRs anyway I created this check to fail the action when triggered by pull request.

Reply:
Is there a way we can just not run the action from the beginning instead of cancelling it after it was created?